templates/content/basic/src/MyProvider.DesignTime/MyProvider.DesignTime.fsproj

```diff
@@ -1,15 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="..\..\netfx.props" />
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net45</TargetFrameworks>
-    <Optimize>true</Optimize>
-    <Tailcalls>true</Tailcalls>
-    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
```

Comment:
This should remain, and we must have the template pin the FSHarp.Core reference.  It must not float to latest FSharp.Core.

TPDTC must typically load into F# tooling going back one or two years, which may be using an FSHarp.Core of relatively low version, e.g. devenv.exe of VS2019 uses say 4.7.0.0, fsc.exe similar, Ionide uses the same.  So the template should be pining FSHarp.Core to pacakge 4.7.2 or before to ensure the TPDTC is loadable.  If it floats to 5.0.0 or later the component will not be loadable into much F# tooling